.storybook/addons.js

```diff
@@ -1,2 +1,2 @@
-import '@storybook/addon-actions/register';
-import '@storybook/addon-links/register';
+import '@storybook/addon-knobs/register';
+import '@storybook/addon-options/register';
```

Comment:
Sorry, this is where I should have commented about new lines. Please be careful with not adding them.